tests/power_cycle/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Images
+*.svg
```

Comment:
We shouldnt need to do this because all test output files should be saved to `generated_data/test_data`. This filepath is set up to be given by the `tmp_path` fixture. See https://docs.pytest.org/en/8.2.x/how-to/tmp_path.html for usage examples.

Please remove

Reply:
ufff... this took me longer than I care to admit to learn how to use... XD

Reply:
still need to remove this file!

Reply:
I had removed it from the coils branch instead, sorry!